src/splitInteger.test.js

```diff
@@ -2,20 +2,39 @@
 
 const splitInteger = require('./splitInteger');
 
+test(`should return array`, () => {
+  const parts = splitInteger();
+
+  expect(parts)
+    .toBeInstanceOf(Array);
+});
+
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
+  const parts = splitInteger(32, 6);
 
+  expect(parts)
+    .toEqual([5, 5, 5, 5, 6, 6]);
```

Comment:
Read your summary again and check the data you expect to receive.
Change summary of expected result

Reply:
Fixed